hbshelpers/chainedLookup.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * A Handlebars helper that performs multiple lookups in a row.
+ *
+ * @param {Object} context The context to perform lookup on
+ * @param {...string} lookupChain The lookups to perform.
+ * @param {import('handlebars').HelperOptions} options
+ * @returns {any} The result of the lookup
+ */
+module.exports = function chainedLookup(context, ...lookupChain) {
+  let lookupResult = context;
+  for (const lookup of lookupChain.slice(0, -1)) {
+    if (!context || typeof context !== 'object') {
```

Comment:
is this meant to be `!lookupResult`?

Reply:
ah yes good catch thank you 🙏

Reply:
also my unit tests didn't get checked in by accident, added those in!